.goreleaser.yml

```diff
@@ -19,12 +19,13 @@ archives:
     files:
       - LICENSE
       - README.md
-    replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
+    name_template: >-
+      {{- if eq .Arch "darwin" }}Darwin
+      {{- else if eq .Arch "linux" }}Linux
+      {{- else if eq .Arch "windows" }}Windows
+      {{- else if eq .Arch "386" }}i386
+      {{- else if eq .Arch "amd64" }}x86_64
+
```

Comment:
Do we need a final `else` line like `{{- else }}{{ .Arch }}{{ end }}`, according to the [doc](https://goreleaser.com/deprecations/#archivesreplacements)?